src/headers/profile.h

```diff
@@ -74,5 +75,6 @@ Profile profile_init_console();
 Profile profile_init_console_legacy();
 Profile profile_init_desktop();
 
+extern uint8_t profile_active_index;
```

Comment:
This breaks encapsulation principle - it bad practice to expose implementation variable (and for write access no less).

There already is `void profile_set_active(uint8_t)` function, let's add a get function to go with it.